medic/medic-run.js

```diff
@@ -238,6 +241,26 @@ function windowsSpecificPreparation(argv) {
         );
     }
 
+    // start logging
+    var logScriptPath = path.join(platformPath, 'cordova', 'log');
+    if (fs.existsSync(logScriptPath)) {
+        util.medicLog('Running windows log script')
+        logProcess = cp.fork(logScriptPath, [], { silent: true });
+
+        logProcess.stdout.on('data', function (data) {
+            fs.appendFileSync('./out.log', data, { encoding: util.DEFAULT_ENCODING });
+        });
+        logProcess.stderr.on('data', function (data) {
+            util.medicLog('Logging script STDERR: ' + data);
+            fs.appendFileSync('./out.log', data, { encoding: util.DEFAULT_ENCODING });
+
+            // clear logProcess variable if logging process has exited so we won't try to kill it later
+            if (data.indexOf('No log channels enabled. Exiting...') !== -1) {
```

Comment:
Instead of hardcoding this to a particular error message - we should look for the 'close' event on logProcess and set it to null, and log the corresponding error code.